File: test-smoke/src/test/java/science/monke/api/order/SwaggerSmokeTest.java
package science.monke.api.order;

import io.restassured.RestAssured;
import org.junit.jupiter.api.BeforeAll;
import org.junit.jupiter.api.Test;
import science.monke.ConfigLoader;
import science.monke.enitity.Config;

import java.io.IOException;

class SwaggerSmokeTest {

  @BeforeAll
  static void beforeAll() throws IOException {
    RestAssured.enableLoggingOfRequestAndResponseIfValidationFails();

    final Config config = ConfigLoader.load();
    RestAssured.baseURI = config.getBaseUri();
    RestAssured.basePath = config.getBasePath();
  }

  @Test
  void swagger() {
    RestAssured.given().when().get("/swagger").then().statusCode(200);
  }
}

File: test-smoke/src/test/kotlin/science/monke/api/SwaggerSmokeTest.kt
package science.monke.api;

import io.restassured.module.kotlin.extensions.Given
import io.restassured.module.kotlin.extensions.Then
import io.restassured.module.kotlin.extensions.When
import org.junit.jupiter.api.Test
import science.monke.AbstractSmokeTest

class SwaggerSmokeTest : AbstractSmokeTest() {
    @Test
    fun swagger() {
        Given {
            with()
        } When {
            get("/swagger")
        } Then {
            statusCode(200)
        }
    }
}
